packages/uix-host-react/src/components/Extensible.tsx

```diff
@@ -77,33 +77,22 @@ export function Extensible({
   guestOptions,
   runtimeContainer,
   debug,
-  sharedContext = {},
+  sharedContext,
 }: PropsWithChildren<ExtensibleProps>) {
-  const [extensions, setExtensions] = useState({});
+  const hostName = appName || window.location.host || "mainframe";
```

Comment:
I like calling it "mainframe" because it's some great old-school stuff, but I don't think it's a very informative default name. The host isn't necessarily in a frame itself.

I recommend `default-uix-host`